docs/gl_objects/groups.rst

```diff
@@ -402,7 +402,7 @@ Create group push rules (at least one rule is necessary)::
 
     group.pushrules.create({'deny_delete_tag': True})
 
-Get group push rules (returns ``None`` if there are no push rules)::
+Get group push rules::
```

Comment:
We have the same statement for projects in https://github.com/python-gitlab/python-gitlab/blob/main/docs/gl_objects/projects.rst#examples-10. Do you think we need to remove that too?

Likely this was copied from project push rules. Maybe there was a change in the API at some point.

Reply:
I honestly don't know. I haven't checked it  🤷‍♂️

Reply:
But I guess it doesn't hurt to remove it. Even if it does return `None`. Do we need to document that?